vck/src/commonMain/kotlin/at/asitplus/wallet/lib/data/rfc/tokenStatusList/primitives/MissingPayloadClaimException.kt

```diff
@@ -0,0 +1,3 @@
+package at.asitplus.wallet.lib.data.rfc.tokenStatusList.primitives
+
+class MissingPayloadClaimException(val claimName: String) : IllegalStateException("Missing payload claim `$claimName`")
```

Comment:
As far as I can tell, this is only used in the methods `toStatusListTokenPayload()`, which in turn are never used

Reply:
removed